material/1_mon/firststeps/firststeps_handout.qmd

```diff
@@ -74,9 +74,9 @@ myfunction(0,5;keyword1 = "notdefault")
 1. everything before the `;` => positional, after => `kwargs`
 2. List all methods with that function name - returns two functions, due to the `b=123` optional positional argument
 
-:: callout-tip
-## Terminology function vs. method
-Methods are instantiations of a abstract `function`
+::: callout-tip
```

Comment:
if you have `## blabla` immedidiately within an callout-tip, it uses that as a header for the callout-tip rather than making a headline